DFE-Digital/Dfe.Academies.External
# 108805 remove formasat application type

see below ticket for list of changes:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/108805?McasTsid=26110

## Type of change

## Expected behaviour
Amended text on following 2 pages as per figma to remove wording about form a sat:-
- New Application - what are you applying to do
- Home / landing page - what you will need link - what you will need page

Also, removed tests around form a sat, hence why 8 files changed rather than 2

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. Browse to New Application - what are you applying to do - formasat radio button no longer visible
2. Browse to Home / landing page - what you will need link - what you will need page - form a sat bullet no longer visible

## Prerequisites
n/a
